assets/js/search.js

```diff
@@ -73,6 +73,11 @@
         if (query.length < 3) return hideResults(true);
         if (query === lastQuery) return;
 
+        recordEvent('search', {
```

Comment:
Should we make `search` and `view_search_results` as constants like `CLASSNAME_SPINNING` above?

Reply:
Being GA's keywords that will never change, used more than once, and need not be shared with another component, I'd say it is wasteful to const them.